plugins/techdocs/src/reader/components/TechDocsReaderPageHeader/TechDocsReaderPageHeader.tsx

```diff
@@ -133,7 +133,9 @@ export const TechDocsReaderPageHeader = (
           }
         />
       )}
-      {lifecycle ? <HeaderLabel label="Lifecycle" value={lifecycle} /> : null}
+      {lifecycle ? (
```

Comment:
The [logical and operator](https://react.dev/learn/conditional-rendering#logical-and-operator-) might make sense here.
`  {lifecycle && (<HeaderLabel label="Lifecycle" value={String(lifecycle ?? '')} />) }`

Reply:
Not quite, wouldn't want to for example render an empty string if that's what the lifecycle is

Reply:
I don't think that's how the logical and operator functions 
https://codesandbox.io/s/vigilant-boyd-86pxq3?file=/src/App.js

Reply:
This might work better since `String(lifecycle ?? '')` won't ever be reached if 
 lifecycle does not pass the initial check
`{lifecycle && (<HeaderLabel label="Lifecycle" value={lifecycle} />) }`

Reply:
@chalkedgoose try experiment a bit in a console with different values before and after `&&`

Reply:
Ah React will ignore the falsy values though, so ends up working. Anyway, no strong opinions on the style here, feel free to pick either

Reply:
In the React documentation, the only issue we should be concerned about is the handling of number primitives which it does not look like this code is dealing with. If you have specific examples or console output you'd like to discuss, please share them; otherwise, using this operator in JSX appears to be a good choice for reducing code redundancy. I'm open to further discussion I'm not trying to be stubborn or a hardhead but it seems like a straightforward improvement overall and in the context of this PR no downsides.
<img width="944" alt="image" src="https://github.com/backstage/backstage/assets/26678464/e55a4659-875f-48e1-8586-9c95423afb43">

Reply:
I agree with @chalkedgoose, we should short circuit the component render cycle since we are checking falsy values.